packaging/windows/netdata.wxs

```diff
@@ -0,0 +1,72 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+    <Package Name="Netdata Agent Installer"
+             Manufacturer="Netdata Inc."
+             Version="2.0.0.0"
```

Comment:
I suggest our `netdata.wxs` to be a `netdata.wxs.in` and we use Cmake to overwrite the version.